python/iwp/data_loader.py

```diff
@@ -513,3 +515,64 @@ def add_timestep_as_coord( ds ):
                             preprocess=add_timestep_as_coord )
 
     return ds
+
+
+    def get_data_loader( dataset_path_pattern,
+                         time_indices,
+                         variables=[],
+                         xy_slice_indices=[],
+                         batch_size=1000,
+                         shuffle=True,
+                         num_workers=1 ):
+        """
+        Initializes an DataLoader object that has been seeded with an 
```

Comment:
Nit: The column this is wrapped at is inconsistent with the argument description below.  I'd aim for ~80 columns (not hard until 100+) of what is printed via help() rather than in the code itself - that is, the actual line can run ~85 columns since it is in an indented docstring.